docs/Client_Libraries/go.mdx

```diff
@@ -182,3 +176,69 @@ func main() {
     }
 }
 ```
+
+### Pull a docker or OCI image from a remote repository
+You can also use an OCI layout folder to store docker/OCI images:
+
+```go
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"oras.land/oras-go/v2"
+	"oras.land/oras-go/v2/content/oci"
+	"oras.land/oras-go/v2/registry/remote"
+	"oras.land/oras-go/v2/registry/remote/auth"
+	"oras.land/oras-go/v2/registry/remote/retry"
+)
+
+func pullImage() error {
+	// 0. Create an oci store
+	o, err := oci.New("/tmp/")

```

Comment:
nit:
```suggestion
	// 0. Create an OCI store
	store, err := oci.New("/tmp/myrepo")
```
?

Reply:
Good catch. If we are going to be specific about how the folder will be used, would `oci-layout-root` be a better name?

Reply:
sounds good

Reply:
@qweeah Could you resolve this conversation as completed?

Reply:
This was resolved comment by @Wwwsylvia with lgtm